_posts/2021-04-26-4.0.0-release.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Apache Arrow 4.0.0 Release"
+date: "2021-04-26 00:00:00 -0600"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 4.0.0 release. This covers
+3 months of development work and includes [**711 resolved issues**][1]
+from [**114 distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 3.0.0 release, Yibo Cai has been invited as a committer to Arrow, and Andrew Lamb and Jorge Leitão have joined the Project Management Committee (PMC). Thank you for all of your contributions!
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
```

Comment:
```suggestion

The Arrow C++ library now includes a [vcpkg.json](https://github.com/apache/arrow/blob/master/cpp/vcpkg.json) manifest file and a new CMake option `-DARROW_DEPENDENCY_SOURCE=VCPKG` to simplify installation of dependencies using the [vcpkg](https://github.com/microsoft/vcpkg) package manager. This provides an alternative to using conda for installing C++ library dependencies. See [Developing on Windows](https://arrow.apache.org/docs/developers/cpp/windows.html) for details.
```

Reply:
I thought vcpkg wasn't just for Windows?

Reply:
Correct, it works on Linux and macOS too, but its value is greatest to Windows users, so I thought I'd just emphasize that here. But I guess it makes sense to describe it as a cross-platform solution, so I modified the text to do that.